src/main/java/mate/academy/store/controller/BookController.java

```diff
@@ -34,12 +35,13 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    public BookDto createBook(@RequestBody CreateBookRequestDto requestDto) {
+    public BookDto createBook(@RequestBody @Valid CreateBookRequestDto requestDto) {
         return bookService.save(requestDto);
     }
 
     @PutMapping("/{id}")
-    public BookDto updateBook(@PathVariable Long id, @RequestBody CreateBookRequestDto requestDto) {
+    public BookDto updateBook(@PathVariable Long id,
```

Comment:
```suggestion
    public BookDto updateBook(@PathVariable @Positive Long id,
```

Add `@Validated` under the controller
Also, check other PathVariable